docs/user_guide/mitigation.rst

```diff
@@ -649,17 +649,9 @@ Exponentiated Gradient
 Grid Search
 ~~~~~~~~~~~
 
-.. topic:: References:
+.. _references:
```

Comment:
Is this smart enough to include only the cited ones on this page? It will be important moving forward when there are perhaps dozens of citations across the website, but only say, a handful, on this particular page.

Reply:
I mirrored what we already had in the documentation. Right now the structure is - Each page like `mitigation.rst` will only list citations that are used on this page.

Reply:
That sounds great. I'm pretty sure that's what @romanlutz had in mind too.